pom.xml

```diff
@@ -7,23 +7,29 @@
     <description>Replication Task Suite: A DSpace add-on providing Curation Tasks that perform replication
      (backup and restore) of DSpace content to other locations or services.</description>
     <url>https://wiki.lyrasis.org/display/DSPACE/ReplicationTaskSuite</url>
-    <version>6.3-SNAPSHOT</version>
+    <version>7.0-SNAPSHOT</version>
 
     <organization>
         <name>LYRASIS</name>
         <url>http://www.dspace.org</url>
     </organization>
 
+    <parent>
+        <groupId>org.dspace</groupId>
+        <artifactId>modules</artifactId>
+        <version>7.0-beta5</version>
+    </parent>
+
     <properties>
         <!-- DSpace Version Information (supported version of DSpace) -->
-        <dspace.version>[6.0,6.20)</dspace.version>
+        <dspace.version>7.0-beta5</dspace.version>
         <!-- DuraCloud Version Information (supported version of DuraCloud) -->
         <duracloud.version>6.1.1</duracloud.version>
```

Comment:
The DuraCloud version should also be bumped to 7.0.0, as earlier versions aren't guaranteed to work with Java 11. As part of this, the exclusions for slf4j could be reviewed, as DSpace 7 no longer includes the xmlui.

Reply:
I was wondering about the DuraCloud version too. Thanks, Bill. Updated and dropped slf4j exclusions as well, as they no longer appear to have conflicts.

Note in the latest push, I also reverted my previous change to set the parent because it turned out to cause build problems with building in isolation, details in commit message.